process/02-onboarding/stakeholder-presentation-en.md

```diff
@@ -0,0 +1,78 @@
+build-lists: true
+
+# Stakeholder Onboarding
+
+^ Use "Sketchnote" theme
+
+---
+
+# Agenda
+
+- The nature of software development
+- Risks in developing software
+- Practices we embrace
+
+---
+
+> Software creation is an empirical activity
```

Comment:
Do we need to example on that more? Here are some notes I have:

# Why the pain?

* Waterfall is the wrong process
* Suited for manufacturing (planned process)
* Not suited for new product development (empirical process)

^ The empirical model of process control provides and exercises control through frequent inspection and adaptation for processes that are imperfectly defined and generate unpredictable and unrepeatable outputs. 

# Empirical Process #

## "imperfectly defined and generate unpredictable and unrepeatable outputs"

# Empirical Process (new product development) #